### PR TITLE
Pipeline for building tidy df for indicators 

### PR DESCRIPTION
**Pull request by damianooldoni**
The Rmd takes care of:

1. retrieving taxa from checklists
2. retrieving taxonomic information for taxa in 1. which match GBIF backbone
3. retrieving distribution information for each taxa
4. retrieving description information for each taxa
5. merge all retrieved information
6. tidying
7. filtering alien species based on `establishmentMeans` (see #28), `occurrenceStatus` (see #26) and `country`  equal to `BE` (at the moment the problem with synonyms is not tackled, see https://github.com/trias-project/pipeline/issues/21#issuecomment-382038846)
7. saving final df as `.tsv` file

The data.frame output is now 100% tidy (see https://github.com/trias-project/pipeline/issues/30), in particular the columns `type` and `description` don't exist anymore, the 5 different types are *spread* in 5 columns:
1. `pathway`
2. `native range`
3. `origin`
4. `invasion stage`
5. `habitat`

### CODE REVIEW

**Comment by peterdesmet** [comment]
At some point, a large part of this functionality will have to be ported to https://github.com/trias-project/unified-checklist, but it doesn't have to be now.

File: src/build_checklist_indicators.Rmd [hunk 1]
```diff
@@ -75,57 +75,209 @@ taxon <- trias::gbif_get_taxa(
...
 
+### Distribution
+
 Extract distribution information
 
 ```{r retrieve_distribution_information}
-print("write here your code, Damiano!")
+distribution <- data.frame(key = taxon_keys) %>%
+  rowwise() %>%
+  do_(interp(~ as.data.frame(rgbif::name_usage(key = .$key,
```
Comment (stijnvanhoey) [inline src/build_checklist_indicators.Rmd]: any reason that here `rgbif` is explicitly mentioned and on line 84 not?

File: src/build_checklist_indicators.Rmd [hunk 2]
```diff
@@ -75,57 +75,209 @@ taxon <- trias::gbif_get_taxa(
...
+                                               return = "data",
+                                               data = "distribution"))))
+distribution %<>% select(-source)
```
Comment (stijnvanhoey) [inline src/build_checklist_indicators.Rmd]: maybe just append the `select` to the previous statement after the `do`?

Comment (damianooldoni) [reply src/build_checklist_indicators.Rmd]: Yes, you are right. Just a reminder of previous testing...
Done here 09850a8

File: src/build_checklist_indicators.Rmd (hunk 1 above)
Comment (damianooldoni) [reply src/build_checklist_indicators.Rmd]: no reason. So, I added `rgbif::` on line 84 
Done here e67ec30

**Pull request merged**